subprojects/common/files/src/main/kotlin/com/avito/utils/Path.kt

```diff
@@ -1,12 +1,13 @@
 package com.avito.utils
 
+import androidx.annotation.RequiresApi
 import com.avito.android.Result
 import java.nio.file.Path
 
 /**
  * WARNING: only android 26+ supported
  */
-@Suppress("NewApi")
+@RequiresApi(api = 26)
 fun Path.deleteRecursively(): Result<Unit> {
```

Comment:
So what will I use at API lower than 26?

Reply:
We don't provide a replacement because we don't have such cases.
The point was only to show the restriction explicitly.

Reply:
nothing, this method is a helper for gradle runtime, but nothing will stop you on using it from android because `common:files` comes as transitive dependecy

This check could help you avoid runtime exception